llvm/lib/Transforms/Utils/Evaluator.cpp

```diff
@@ -124,6 +125,20 @@ isSimpleEnoughValueToCommit(Constant *C,
   return isSimpleEnoughValueToCommitHelper(C, SimpleConstants, DL);
 }
 
+static std::pair<bool, GlobalValue::LinkageTypes>
+getPruningAndLinkage(const GlobalVariable &GV) {
+  if (const GlobalObject *GO = GV.getBaseObject()) {
```

Comment:
This is the only non-const pointer in this group is there a reason? (It would be good to be consistent.)

Reply:
Fixed, changed to const pointer.